wafer/users/views.py

```diff
@@ -53,3 +56,11 @@ class EditProfileView(EditOneselfMixin, UpdateView):
 
     def get_success_url(self):
         return reverse('wafer_user_profile', args=(self.object.user.username,))
+
+
+class UserViewSet(viewsets.ModelViewSet):
+    """API endpoint for users."""
+    queryset = get_user_model().objects.all()
+    serializer_class = UserSerializer
+    # We want some better permissions than the default here, but
+    # IsAdminUser will do for now.
```

Comment:
It's inheriting the default permission from settings. It's probably not a bad idea to set it explicitly here (and for the Schedule REST stuff, where the same consideration applies).